### ISSUE TITLE
[BUG] `Azure Endpoint` is a wrong name in settings

### ISSUE THREAD

**Issue by ifsheldon** [Bin-Huang/chatbox#952]
**Bug Description**
In Setting, `Azure Endpoint` is actually `api_base`. When a real endpoint link is used, Chatbox gives me 404 from Azure OpenAI. Please change the name to avoid confusion.

**Steps to Reproduce**
Please provide the steps to reproduce the bug:

1. Go to "Setting"
2. Click on "Model/AI Model Provider"
3. Scroll down to "Azure OpenAI API"
4. Enter an endpoint link given by Azure OpenAI Studio and everything else
5. Ask a question in chat
6. Get 404 from Azure OpenAI

**Expected Results**
Valid response, not 404

**Actual Results**
404 response from Azure OpenAI

**Screenshots**
<img width="749" alt="image" src="https://github.com/Bin-Huang/chatbox/assets/39153080/d6344047-512c-4f79-b2b1-d0dc807dd313">

**Desktop (please complete the following information):**

-   Operating System: macOS
-   Application Version: 1.0.0

**Additional Context**
The actual endpoint is composed like this
```
https://<resource_name>.openai.azure.com/openai/deployments/<deployment_name>/chat/completions?api-version=<api_version>
```
but a `api_base` is just `https://<resource_name>.openai.azure.com/`. When I use `api_base` and enter it into `Azure Endpoint` box, I got results fine.

**Comment by Bin-Huang** [comment]
![截屏2023-09-14 09 58 18](https://github.com/Bin-Huang/chatbox/assets/20723142/765c384e-05c9-44c4-b8ac-5215f1f1009f)
I think on the Azure console page, the Endpoint should refer to the address in the picture. That's how I understand it, and I would like to hear your different points of view.

**Comment by ifsheldon** [comment]
<img width="2089" alt="image" src="https://github.com/Bin-Huang/chatbox/assets/39153080/8905bb51-5cb7-49e0-91eb-e0fbb397e8f5">

OK, then that's Azure's fault. There is an inconsistency where Azure guys define an endpoint differently, which is sloppy. I got my endpoint from Azure OpenAI Studio while yours comes from Azure resource management. 

Probably we can use regex to determine which one a link is, then either one works nonetheless.

**Comment by Bin-Huang** [comment]
This has been fixed in version v1.0.2 
https://github.com/Bin-Huang/chatbox/releases
